### PR TITLE
Adding blur event to k-checkbox

### PR DESCRIPTION
**Pull request by DXCanas**
Needed for validation! Arised inside the vf plugin

### CODE REVIEW

**Comment by DXCanas** [comment]
Dunno if this should go in 6.1 or develop

File: kolibri/core/assets/src/views/k-checkbox/index.vue [hunk 1]
```diff
@@ -137,6 +137,10 @@
           this.$emit('change', this.isCurrentlyChecked, event);
         }
       },
+      markInactive() {
+        this.isActive = false;
+        this.$emit('blur');
```
Comment (christianmemije) [inline kolibri/core/assets/src/views/k-checkbox/index.vue]: Add event description.

**Comment by christianmemije** [comment]
@DXCanas Target 0.6.1

File: kolibri/core/assets/src/views/k-checkbox/index.vue (hunk 1 above)
Comment (christianmemije) [reply kolibri/core/assets/src/views/k-checkbox/index.vue]: :pray: :) (I need to sound nicer.)

Comment (DXCanas) [reply kolibri/core/assets/src/views/k-checkbox/index.vue]: :smirk: thanks lol

What're these for? Do they get included in documentation?

Comment (christianmemije) [reply kolibri/core/assets/src/views/k-checkbox/index.vue]: For the style guide documentation.

**Comment by DXCanas** [comment]
@christianmemije added documentation and rebased

**Pull request merged**